conferences/migrations/2_create_ticketing_tables.up.sql

```diff
@@ -0,0 +1,65 @@
+CREATE TABLE slot_claim (
+    id BIGSERIAL PRIMARY KEY,
+    conference_slot_id BIGINT, 
+    ticket_id VARCHAR(100) CONSTrAINT ticket_id_is_unique UNIQUE,
+    redeemed BOOLEAN,
+    FOREIGN KEY(conference_slot_id) REFERENCES conference_slot(id)
+);
+
+CREATE TABLE attendee (
+    id BIGSERIAL PRIMARY KEY,
+    email BIGINT, 
+    coc_accepted BOOLEAN
+);
+
+CREATE TABLE attendee_to_slot_claims (
+    attendee_id BIGINT,
+    slot_claim_id BIGINT CONSTRAINT slot_claim_id_is_unique UNIQUE,
+    FOREIGN KEY (attendee_id) REFERENCES attendee(id),
+    FOREIGN KEY (slot_claim_id) REFERENCES slot_claim(id)
+);
+
+CREATE TABLE claim_payment (
+    id BIGSERIAL PRIMARY KEY,
+    invoice TEXT -- just in case we need to store the whole thing.
+);
+
+CREATE TABLE payment_method_money (
+    id BIGSERIAL PRIMARY KEY,
+    amount INTEGER,

```

Comment:
We were originally using decimal for this and the corresponding go type but oto did not like it at all so we decided to try our hand at ints for money, since oto is out of the picture we could go back to decimals i think. @bketelsen ?